src/main/java/com/orbit/frc/subsystems/leds/LedsConfig.java

```diff
@@ -0,0 +1,18 @@
+package main.java.com.orbit.frc.subsystems.leds;
+
+import com.revrobotics.CANSparkBase;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkFlex;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj.AnalogEncoder;
+
+import com.orbit.frc.subsystems.SubsystemConfig;
+
+public class LedsConfig extends SubsystemConfig{
```

Comment:
not sure we need it to extend subsystem config actually... might make more sense to avoid doing that cuz theres no motors here